### DISCUSSION TITLE
Customize ingredient note colors for better clarity in cooking mode.

### DISCUSSION THREAD

**Discussion by JiangLongLiu** [mealie-recipes/mealie#3676] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

The notes in the ingredient list can be highlighted in different colors, like gray or green, while you're in cooking mode. You can even customize these colors! This makes the ingredient list clearer and simpler to follow at each step.

### Please provide a concise description of the feature that would resolve your issue.

Same as above

### Please consider and list out some caveats or tradeoffs made in your design decision

Same as above

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
Are you able to provide a bit of a mock up of what you think this might look like please? I'm a bit unclear what it'd be.

**Reply by JiangLongLiu** [reply]
As you can see in the picture, I've marked the ingredients in green and the notes in orange. Before highlighting them, everything was in black text, making it hard to distinguish in cooking mode.

![Screenshot_2024-06-02-01-50-26-902_com android chrome-edit](https://github.com/mealie-recipes/mealie/assets/7019907/37eab459-2a9c-48e0-bccf-7c0f02ab6845)
